README.md

```diff
@@ -116,32 +116,76 @@ On OS X:
+2. In the newly created nyc-council-councilmatic repo, **create** a core file: `touch nyc-council-councilmatic/core.properties` (This file helps solr discover cores for multicore processing. That way, one Solr installation can run multiple apps.)
+3. Solr expects to find several files in the conf repo of Councilmatic. We'll use the example conf (`sample_techproducts_configs`) and parse it down. **Copy** the contents of `solr/configsets/sample_techproducts_configs/` into `solr/nyc-council-councilmatic`: `cp -R configsets/sample_techproducts_configs/* nyc-council-councilmatic/`.
+
+Note: you do not need all the files provided in `sample_techproducts_configs`. You can safely remove several, including: _rest_managed.json, _schema_analysis_stopwords_english.json, _schema_analysis_synonyms_english.json, mapping-FoldToASCII.txt, mapping-ISOLatin1Accent.txt, update-script.js.
+
+Next, you need to set-up the NYC Councilmatic schema. We use a [classic schema file](https://lucene.apache.org/solr/guide/7_1/schema-factory-definition-in-solrconfig.html#switching-from-managed-schema-to-manually-edited-schema-xml). First, open `solrconfig.xml`, which lives inside `/solr/nyc-council-councilmatic/conf/`. Anywhere between the `<config>` tags, add the following:
 
-# Copy schema.xml for this app to solr directory
-sudo cp solr_scripts/schema.xml /opt/solr/solr/collection1/conf/schema.xml
+```
+<schemaFactory class="ClassicIndexSchemaFactory"/>
 ```
 
-**Run Solr**
-```bash
-# Next, start the java application that runs solr
-# Do this in another terminal window & keep it running
-# If you see error output, somethings wrong
-cd /opt/solr
-sudo java -jar start.jar
+Then, rename the managed-schema file to schema.xml:
+
+```
+cd /solr/nyc-council-councilmatic/conf
+mv managed-schema schema.xml
 ```
 
-**Index the database**
-```bash
-# back in the nyc-councilmatic directory:
+The solr_scripts repo in NYC Councilmatic contains a working schema.xml. Copy the contents of this file into `/solr/nyc-council-councilmatic/conf/schema.xml`.
+
+In `settings_deployment.py`, be sure to include the correct URL in HAYSTACK_CONNECTIONS. The URL should align with the name of the solr core you created above:
+
+```
+HAYSTACK_CONNECTIONS = {
+    'default': {
+        'ENGINE': 'haystack.backends.solr_backend.SolrEngine',
+        #'URL': 'http://127.0.0.1:8983/solr'
+        # ...or for multicore...
+        'URL': 'http://127.0.0.1:8983/solr/nyc-council-councilmatic',
+    },
+}
+```
+
+Finally, build your index:
+
+```
 python manage.py rebuild_index
 ```
 
+Is your solr running? Go to `/solr-7.1.0/bin`, and execute:
+
+```
+./solr start
+```
+
+Then, visit [http://127.0.0.1:8983/solr/#/](http://127.0.0.1:8983/solr/#/), and see solr in action. If you need to stop solr, do the following:
+
+```
+/solr stop -p 8983
+```
+
 **OPTIONAL: Install and configure Jetty for Solr**
```

Comment:
The way that the newer Solr stuff works, this is no longer needed. It actually uses jetty (or something similar) to run the application.